src/renderer/components/common/SelectableSignatory/SelectableSignatory.tsx

```diff
@@ -46,12 +45,12 @@ const SelectableSignatory = <T extends any>({
     >
       <AccountAddress addressFont="text-body text-inherit" size={size} name={name} {...addressProps} />
       <InfoPopover data={popoverItems}>
-        <Icon name="info" size={14} className="text-icon-default ml-2 mr-auto" />
+        <Icon name="info" size={14} className="ml-2 mr-auto" />
       </InfoPopover>
       {balance && asset && (
         <BalanceNew value={transferableAmount(balance)} asset={asset} className="text-body text-inherit ml-auto mr-6" />
       )}
-      <Icon name="right" className={cnTw('text-icon-default', !balance && 'ml-auto')} size={20} />
+      <Icon name="right" className={balance ? '' : 'ml-auto'} size={20} />
```

Comment:
Can use `cnTw` instead of ternary operator
```suggestion
      <Icon name="right" className={cnTw(!balance && 'ml-auto')} size={20} />
```

Reply:
done